docs/avs/operator-onboarding/prerequisites.md

```diff
@@ -11,57 +11,64 @@ Before starting the [onboarding process](./onboard/) using the [DIN app](https:/
 Compatible wallets include MetaMask and WalletConnect. DIN supports
 [multiple wallet setup options](./mainnet-preparation.md#wallet-setup-options).
 
+### Mainnet
+
+- Ensure you have a **brand new externally owned account (EOA) wallet**, never used with any other EigenLayer AVS.
+- Ensure your wallet is **dedicated to DIN only**.
+  It cannot be shared with other EigenLayer AVS operations.
+
 :::danger Critical for Mainnet
 Your wallet MUST be dedicated exclusively to DIN AVS. Using a wallet associated with another AVS will cause operational failures.
 :::
 
-### Mainnet
-
-- [ ] **Brand new externally owned account (EOA) wallet** - Never used with any other EigenLayer AVS
-- [ ] **Dedicated to DIN only** - Cannot be shared with other EigenLayer AVS operations
```

Comment:
Note: these checkboxes are not clickable in the rendered site, so I changed them to regular list items.